test/e2e/multimodule/e2e_suite_test.go

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
@jakobmoellersap, 

For the new plugin test be called we need to add it in : https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/e2e/setup.sh#L65